app/src/main/java/org/mozilla/fenix/tabtray/TabTrayView.kt

```diff
@@ -659,6 +659,10 @@ class TabTrayView(
                 1
```

Comment:
I am wondering if we can remove this "1" that I added for displaying the tab item above the selected browser tab.